src/Microsoft.Framework.Caching.Redis/CachingServiceExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Caching.Distributed;
+using Microsoft.Framework.Caching.Redis;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class CachingServicesExtensions
+    {
+        public static IServiceCollection AddRedisCache(this IServiceCollection collection)
+        {
+            collection.AddOptions();
+            collection.TryAdd(ServiceDescriptor.Transient<IDistributedCache, RedisCache>());
```

Comment:
We do the same thing here: https://github.com/aspnet/Caching/blob/dev/src/Microsoft.Framework.Caching.Distributed/CachingServicesExtensions.cs#L14
This needs to be worked out across all of caching. It's not a problem for this PR.